cli/internal/cmd/verify.go

```diff
@@ -124,20 +124,20 @@ func (v *verifyCmd) parseVerifyFlags(cmd *cobra.Command, fileHandler file.Handle
 	if err != nil {
 		return verifyFlags{}, fmt.Errorf("parsing config path argument: %w", err)
 	}
-	v.log.Debugf("configuration: %s", configPath)
+	v.log.Debugf("Configuration file flag is %s", configPath)
 
 	ownerID := ""
 	clusterID, err := cmd.Flags().GetString("cluster-id")
 	if err != nil {
 		return verifyFlags{}, fmt.Errorf("parsing cluster-id argument: %w", err)
 	}
-	v.log.Debugf("cluster-id: %s", clusterID)
+	v.log.Debugf("Cluster ID flag is %s", clusterID)
 
 	endpoint, err := cmd.Flags().GetString("node-endpoint")
 	if err != nil {
 		return verifyFlags{}, fmt.Errorf("parsing node-endpoint argument: %w", err)
 	}
-	v.log.Debugf("node-endpoint: %s", endpoint)
```

Comment:
Imo this isn't a good idea, as it is not as precise as before. Node endpoint flag could also be the endpoint flag of the node. Maybe something like "Flage 'node-endpoint' is %q"?

Reply:
Let's just go with "'node-endpoint' flag is %q" so every sentence has flag at the end?

Reply:
Fine for me, but I'd also apply this to the other flags you've changed above.

Reply:
ok, then let's just leave it like this.